scope flows crashes for big payloads

```
root@14cd87fb734d:/# scope run -p -- apt upgrade
(...)
Do you want to continue? [Y/n] 
(...)
Processing triggers for libc-bin (2.31-0ubuntu9.2) ...
root@14cd87fb734d:/# scope flows 
panic: close of closed channel

goroutine 20 [running]:
github.com/criblio/scope/flows.getFlowEvents.func1(0xc00024f380, 0x2, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/runner/work/appscope/appscope/cli/flows/flow.go:150 +0xba
created by github.com/criblio/scope/flows.getFlowEvents
	/home/runner/work/appscope/appscope/cli/flows/flow.go:146 +0x16a
root@14cd87fb734d:/# scope history
Displaying last 20 sessions
ID	COMMAND	CMDLINE    	PID	AGE  	DURATION	TOTAL EVENTS 
1 	apt    	apt update 	10 	1m26s	2s      	677         	
2 	apt    	apt update 	247	1m14s	270ms   	444         	
3 	apt    	apt update 	485	57s  	526ms   	452         	
4 	apt    	apt upgrade	735	23s  	0ms     	2321        	
root@14cd87fb734d:/# scope flows -i 4
panic: close of closed channel

goroutine 7 [running]:
github.com/criblio/scope/flows.getFlowEvents.func1(0xc0002353a0, 0x2, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/runner/work/appscope/appscope/cli/flows/flow.go:150 +0xba
created by github.com/criblio/scope/flows.getFlowEvents
	/home/runner/work/appscope/appscope/cli/flows/flow.go:146 +0x16a
```

reproduced on cribl/scope:0.6.0-rc15

Doesn't affect me in any way, but FTR I can still repro on 1.1.3:
```
~ alias sc
alias sc='sudo ~jb/bin/scope'
~ sc run -p -- apt update
WARNING: Session history will be stored in /root/.scope/history and owned by root
(...)
~ sc run -p -- apt upgrade
WARNING: Session history will be stored in /root/.scope/history and owned by root
(...)
~ sc history
(...)
115	apt    	apt update         	646928 	2m58s	49s     	1323
116	apt    	apt upgrade        	650060 	2m0s 	0ms     	10949

~ sc flows -i 115 # ok!
ID    	HOST IP      	HOST PORT	PEER IP        	PEER PORT	LAST SENT	DURATION
zLNTX3	192.168.1.104	39226    	151.101.2.217  	80       	3m38s    	0s
(...)
```

The difference now is that flows for apt upgrade (-i 116) now behaves nondeterministically
```
~ sc flows -i 116 # sometimes it works
ID    	HOST IP      	HOST PORT	PEER IP        	PEER PORT	LAST SENT	DURATION
DTOLm2	192.168.1.104	38180    	142.250.203.142	80       	2m6s     	0s
(...)
~ sc flows  # sometimes it panics
panic: close of closed channel

goroutine 7 [running]:
github.com/criblio/scope/flows.getFlowEvents.func1()
	/home/builder/appscope/cli/flows/flow.go:161 +0x11a
created by github.com/criblio/scope/flows.getFlowEvents
	/home/builder/appscope/cli/flows/flow.go:154 +0x1d5
	
~ sc flows -i 116 # sometimes it throws an error
error getting flows: error reading events: bufio.Scanner: token too long
```

The difference might be not due to software change but a different payload (that `apt upgrade` downloaded google-chrome-stable if it matters)

--- a/cli/util/newlinereader.go
+++ b/cli/util/newlinereader.go
@@ -12,6 +12,8 @@ import (
 func NewlineReader(r io.Reader, match func(string) bool, callback func(line int, offset int64, b []byte) error) (int, error) {
 	cr := &CountingReader{Reader: r}
 	scanner := bufio.NewScanner(cr)
+	buf := make([]byte, 0, 64*1024)
+	scanner.Buffer(buf, 1024*1024)
 	idx := 0
 	offset := int64(0)
 	for scanner.Scan() {

